### DISCUSSION TITLE
可以把通知书里面涉及到的各地银行分行支行也做个汇总吗？？

### DISCUSSION THREAD

**Discussion by h9j6k** [WeNeedHome/SummaryOfLoanSuspension#840] [Ideas]
你好，

停贷通知书上附带有监管不利的银行的名单，所以

１．能不能把监管不利的银行也做个汇总？？

２．把涉及到的银行也在地图上做个标记？？？

可以帮助储户避雷，谢谢。

**Comment by MarkShawn2020** [comment]
这个需要ocr支持，工作量比较大

**Reply by h9j6k** [reply]
多谢考虑，可不可以这样，已经提交的图片将来有富余的精力用ｏｃｒ去识别，新提交的图片，在ｐｒ的时候提示提交用户用文字整理图片里面的银行列表，这样新增的楼盘就可以有银行的信息。

**Reply by MarkShawn2020** [reply]
这个需要用户达成共识，目前仓库热度已经逐渐降低，我认为可能配合的人不会太多？因为不单单是银行，还有开发商，人工操作工作量并不少

**Reply by h9j6k** [reply]
嗯，无奈我不是程序员的，要是能用ｏｃｒ帮到你们就好了。但是我看到你们这么忙前忙后，确实很感动，大家加油！

**Reply by MarkShawn2020** [reply]
> 嗯，无奈我不是程序员的，要是能用ｏｃｒ帮到你们就好了。但是我看到你们这么忙前忙后，确实很感动，大家加油！

我加入后端todo吧
